### ISSUE TITLE
Inconsistent render behavior between editor and WebGL implementation

### ISSUE THREAD

**Issue by tshadowknight** [effekseer/Effekseer#1015]
Hi,

I've been using Effekseer in a web project for a while now and things have been working fine but I have run into an issue.
The issue being that for certain combinations of elements with different blend modes the end result is different between the editor and the runtime.

I have an example effect set up that highlights my problem: [subtract_test.zip](https://github.com/user-attachments/files/15904213/subtract_test.zip)

The setup is:
-one pane with blend mode blend, blue in example
-one pane with blend mode subtract, pure white in the example(appears black)
-one particle effect with blend mode add, white in the example

In the editor the final result is like this:

![image](https://github.com/effekseer/Effekseer/assets/23508463/e89e2c3a-3d58-47dd-8612-907d12c54492)

The particles viewed through the panes appear blue.

In the runtime the final result is like this:

![image](https://github.com/effekseer/Effekseer/assets/23508463/41c6638e-70d3-44ca-acd8-2264b211eea8)

The particles viewed through the panes appear white.

Since I've been experimenting with using negative blend panes for tinting an effect, having this inconsistency is rather inconvenient.
And for the record the effect achieved in editor seems preferable from my end at least. But the most important issue is the lack of consistency.

I can only confirm this issue exists when using the WebGL runtime so I am unsure if this an issue specific to that runtime or a core issue.

Kind regards,
TSK

**Comment by tshadowknight** [comment]
An example of the issue with a more complex effect for reference:

![image](https://github.com/effekseer/Effekseer/assets/23508463/52d38aab-94b1-44ae-9b64-eb4849e44dc5)

**Comment by durswd** [comment]
Subtract is complicated. It depends many things

**Comment by tshadowknight** [comment]
Right, so I suppose it'll be better to try and work around it by using non-subtract blend.
